contracts/HtsSystemContract.sol

```diff
@@ -593,6 +584,19 @@ contract HtsSystemContract is IHederaTokenService {
                 _setApprovalForAll(from, to, approved);
                 return abi.encode(true);
             }
+            if (selector == this.getNonFungibleTokenInfo.selector) {
+                require(msg.data.length >= 92, "getNonFungibleTokenInfo: Not enough calldata");
+                uint256 serialId = uint256(bytes32(msg.data[60:92]));
+                NonFungibleTokenInfo memory nonFungibleTokenInfo;
+                (int64 creationTime, bytes memory metadata) = __nftInfo(serialId);
+                nonFungibleTokenInfo.creationTime = creationTime;
```

Comment:
nit: let's use the same ordering as in `NonFungibleTokenInfo` so it's easier to see that all fields are being set.